frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.stories.tsx

```diff
@@ -2,74 +2,54 @@ import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { fakeUsers } from "@common/fakeData";
 import fakeExperiences from "@common/fakeData/fakeExperiences";
+import { JobLookingStatus } from "@common/api/generated";
 import { ProfilePage, ProfileForm } from "./ProfilePage";
 import { User } from "../../../api/generated";
 
-const fakeUserArray = fakeUsers(5);
+import MockGraphqlDecorator from "../../../../../.storybook/decorators/MockGraphqlDecorator";
+
+const fakeUserData = fakeUsers(1)[0];
+const fakeExperienceArray = fakeExperiences(3);
 
 export default {
   component: ProfilePage,
   title: "Profile Form",
   args: {},
+  decorators: [MockGraphqlDecorator],
```

Comment:
We could just add this decorator globally, if we wanted to disable all attempts to reach our graphql endpoints within stories

(if we do this, our urql hooks will always receive responses of `{data: null}`, if not set more specifically via storybook parameters)

Reply:
Decided to do this. No need for ever having urql hooks calling api in stories.